### DISCUSSION TITLE
Do you implement color mask inside StrongSort?

### DISCUSSION THREAD

**Discussion by tgrmev** [mikel-brostrom/yolo_tracking#671] [Q&A]
Hello, Thank you so much for your repo that I am using. I have a small simulation in Unity and by using unity camera I implement StrongSort on the video taken from Unity camera. Then, I crop the bounding boxed images (detected objects) and implement CNN algorithm to compare them. When I give the images after cropping on Unity video stream, I got the perfect RGB images and CNN works. However, it does not work perfectly on the images cropped after the StrongSort. It looks like masked images. How I convert them to the pure RGB image inside StrongSort or soon after it? To explain my question better, I attached the images.

  
![stronSort_Unity](https://user-images.githubusercontent.com/44428766/210010798-82bdeca7-3008-49d0-9f57-2fbece2364ad.png)

**Comment by mikel-brostrom** [comment]
> However, it does not work perfectly on the images cropped after the StrongSort. It looks like masked images.

I see no segmentation masks on the images...

If you don't want segmentation masks on the output image you can load any model that does not end by `-seg`

**Reply by tgrmev** [reply]
okay, I am gonna try that. So what is the reason of the color differences between input and output of the StrongSORT?

**Reply by mikel-brostrom** [reply]
If you are using a segmentation model we assume you want your images segmented. If you don't then just go with a regular object detection model
